qa/integration/specs/logstash_to_logstash_spec.rb

```diff
@@ -43,7 +43,7 @@ def get_temp_path_dir
     tmp_data_path
   end
 
-  def run_logstash_instance(config_name, options = {})
+  def run_logstash_instance(config_name, options = {}, &block)
     @next_api_port_offset = (@next_api_port_offset||0).next.modulo(1000) # cycle through 1000 possibles
```

Comment:
As we synced up, it would be great if we can set offset to higher value to avoid conflicts iff LS instance appears between.
```suggestion
    @next_api_port_offset = (@next_api_port_offset||100).next.modulo(1000) # cycle through 1000 possibles
```